### PR TITLE
chore: update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

follow the latest convention

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -21,8 +21,4 @@
 - [ ] `.changeset`
 - [ ] unit tests
 - [ ] integration tests
-- [ ] docs
```
Comment (charIeszhao) [inline PULL_REQUEST_TEMPLATE.md:24]: I thought `docs` means we should add or update docs in `https://docs.logto.io`

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by darcyYe — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (gao-sun) [reply PULL_REQUEST_TEMPLATE.md:24]: yes, but the docs update has a really low frequency. would prefer to mention the TSDoc comments instead

Comment (gao-sun) [reply PULL_REQUEST_TEMPLATE.md:24]: synced offline

**Pull request merged**